Source/WebCore/html/HTMLMediaElement.cpp

```diff
@@ -9441,7 +9440,7 @@ void HTMLMediaElement::visibilityAdjustmentStateDidChange()
 void HTMLMediaElement::sceneIdentifierDidChange()
 {
     if (RefPtr page = document().page()) {
-        ALWAYS_LOG(LOGIDENTIFIER, page->sceneIdentifier());
+        HTMLMEDIAELEMENT_RELEASE_LOG(HTMLMEDIAELEMENT_SCENEIDENTIFIERDIDCHANGE, page->sceneIdentifier().utf8().data());
```

Comment:
It is somehow unfortunate to have to call utf8 here?
Ideally, we would do it like ALWAYS_LOG, not sure whether feasible.

Reply:
ALWAYS_LOG is also converting all log arguments to String and utf8:

```
    template<typename... Argument>
    static inline void logVerbose(WTFLogChannel& channel, WTFLogLevel level, const char* file, const char* function, int line, const Argument&... arguments)
    {
        auto logMessage = makeString(LogArgument<Argument>::toString(arguments)...);
        ...
        SUPPRESS_UNRETAINED_LOCAL os_log(channel.osLogChannel, "%{public}s", logMessage.utf8().data());
```

Reply:
`ALWAYS_LOG` and family only call `.utf8().data()` on the composed log string, which is created with `makeString`, so String arguments are not converted.

Reply:
Indirectly, String arguments will be converted, since they are substrings of the String returned by `makeString`, right? Or perhaps there is something I am missing here?

Thanks for reviewing!